features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -276,7 +276,7 @@
   "identity_mgt.user_self_registration.lock_on_creation": true,
   "identity_mgt.user_self_registration.enable_recaptcha": true,
   "identity_mgt.user_self_registration.verification_email_validity": "$ref{identity_mgt.default_mail_validity_period}",
-  "identity_mgt.user_self_registration.callback_url": "${carbon.protocol}://${carbon.host}:${carbon.management.port}/authenticationendpoint/login.do",
+  "identity_mgt.user_self_registration.callback_url": ".*",
```

Comment:
what's the reason to put .* here

Reply:
Otherwise, the callback URL validation will fail in the tenant qualified mode in the self sign up flow. So we allow anything OOTB. But if someone needs to increase security in a production setup they can define a relevant regex pattern that matches their callback URLs.

Reply:
mmm we should match this with default url pattern